csrc/ops/normalization.cpp

```diff
@@ -115,6 +118,11 @@ VarMeanResult variance_mean(
         sub(num_features, IrBuilder::create<Int>(x->container(), correction));
   }
 
+  // Welford op can't handle 0-dim tensors, so we need to handle them separately
+  if (x->nDims() == 0) {
+    return {variance(x, dims, correction, keepdim), mean(x, dims, keepdim)};
```

Comment:
Feels a little bit awkward here, since variance for a scalar tensor is just `nan` and mean is just returning itself. Having said that, we might want to keep it like this in the definition, that feels like an optimization pass's job to simplify that.

Reply:
I agree, but functions in `FusidionDefinition.ops` should still mostly work with edge cases like that. Otherwise, a higher-level project using these functions would need to have more and more wrappers that fix problematic cases.

Reply:
It's not just `nan`, it's also `0.0` 😄 
```py
In [1]: import torch

In [2]: a = torch.randn(())

In [3]: torch.var(a, dim=0, correction=0)
Out[3]: tensor(0.)

In [4]: torch.var(a, dim=0, correction=1)
Out[4]: tensor(nan)
```